modules/fbs/includes/fbs.user.inc

```diff
@@ -62,6 +62,7 @@ function fbs_user_authenticate($name, $pincode) {
     // We have to save the library card number/CPR number, as the pincode
     // change call uses it.
     $result['creds']['library_card_number'] = $name;
+    $result['creds']['pass'] = $pincode;
```

Comment:
I do not think it is a good idea to store the pincode in the creds. It is not required by the integration, the provider in itself or any provider related functionality like changing password mentioned above. It also means that any attacker getting access to the session storage (database) will have usernames and passwords for all users with a session.

On the other hand I do not have any other solution. If we go with this then please add a comment explaining why this is necessary.

@xendk: Do you have any input on this?